php/Terminus/Models/Site.php

```diff
@@ -332,6 +333,19 @@ public function fetch(array $options = array()) {
   }
 
   /**
+   * Re-fetches site attributes from the API
+   *
+   * @return void
+   */
+  public function fetchAttributes() {
+    $response = $this->request->simpleRequest(
+      sprintf('sites/%s/settings', $this->get('id'))
+    );
+    $this->attributes = $response['data'];
+    $this->collection->sites_cache->update((array)$response['data']);
```

Comment:
What are we storing in the cache? I'd prefer for us to touch the cache as little as possible since I'd like to do away with it entirely once we get a sitename->uuid lookup in the api.